src/WalkingModule/src/Module.cpp

```diff
@@ -322,6 +322,14 @@ bool WalkingModule::configure(yarp::os::ResourceFinder& rf)
     m_qDesired.resize(m_robotControlHelper->getActuatedDoFs());
     m_dqDesired.resize(m_robotControlHelper->getActuatedDoFs());
 
+    // TODO please remove me. Counters are evil
+    double stancePhaseTimeOutSeconds = rf.check("stance_phase_time_out",yarp::os::Value(0.5)).asDouble();
+
+    m_stancePhaseMaxCounter = (int) std::round(stancePhaseTimeOutSeconds/m_dT);
+
+    yInfo() << "m_stancePhaseMaxCounter " << m_stancePhaseMaxCounter
```

Comment:
Consider either to make this printout more detailed or to remove it.

Reply:
Removed in https://github.com/robotology/walking-controllers/pull/41/commits/9760be0556d233e940c1faba357e1d48f3f54cf8